src/components/DateRangePickerInput.jsx

```diff
@@ -41,6 +41,7 @@ const propTypes = forbidExtraProps({
   isEndDateFocused: PropTypes.bool,
   showClearDates: PropTypes.bool,
   disabled: PropTypes.bool,
+  selectivelyDisabled: PropTypes.string,
```

Comment:
instead of a string, let's use `PropTypes.oneOf(['none', 'startDate', 'endDate'])`, and let's have that be imported from the same shared module.